Remotion/Data/DomainObjects.UnitTests/Persistence/Rdbms/Parameters/SerializedObjectIDDataParameterDefinitionTest.cs

```diff
@@ -0,0 +1,145 @@
+
+    var serializedObjectIDDataParameterDefinition = new SerializedObjectIDDataParameterDefinition(storageTypeInformation);
+
+    var result = serializedObjectIDDataParameterDefinition.GetParameterValue(objectID);
+
+    Assert.That(result, Is.EqualTo(expectedValue));
+  }
+
+  [Test]
+  public void GetParameterValue_WithNull_ReturnsDBNull ()
+  {
+    var storageTypeInformation = StorageTypeInformationObjectMother.CreateStorageTypeInformation();
+
+    var serializedObjectIDDataParameterDefinition = new SerializedObjectIDDataParameterDefinition(storageTypeInformation);
+
+    Assert.That(serializedObjectIDDataParameterDefinition.GetParameterValue(null), Is.EqualTo(DBNull.Value));
+  }
+
+  [Test]
+  public void GetParameterValue_WithTypeMismatchedValue_ThrowsArgumentException ()
+  {
+    var dummyValue = "Not an ObjectID";
+    var storageTypeInformation = StorageTypeInformationObjectMother.CreateStorageTypeInformation();
+
+    var serializedObjectIDDataParameterDefinition = new SerializedObjectIDDataParameterDefinition(storageTypeInformation);
+
+    Assert.That(
+        () => serializedObjectIDDataParameterDefinition.GetParameterValue(dummyValue),
+        Throws.InstanceOf<ArgumentException>().With.ArgumentExceptionMessageEqualTo(
+            $"Parameter 'value' has type '{typeof(string)}' when type '{typeof(ObjectID)}' was expected.",
+            "value"));
+  }
+
+  [Test]
+  public void CreateDataParameter ()
+  {
+    var commandMock = new Mock<IDbCommand>(MockBehavior.Strict);
+    var dataParameterMock = new Mock<IDbDataParameter>(MockBehavior.Strict);
+
+    var storageTypeInformation = StorageTypeInformationObjectMother.CreateStorageTypeInformation(storageDbType: DbType.AnsiString, storageTypeLength: 255);
+
+    var objectID = DomainObjectIDs.Order1;
+    var testValue = ObjectIDStringSerializer.Instance.Serialize(objectID);
+
+    commandMock.Setup(mock => mock.CreateParameter()).Returns(dataParameterMock.Object).Verifiable();
+
+    dataParameterMock.SetupProperty(mock => mock.DbType);
+    dataParameterMock.SetupProperty(mock => mock.Value);
+    dataParameterMock.SetupProperty(mock => mock.Size);
+
+    var serializedObjectIDDataParameterDefinition = new SerializedObjectIDDataParameterDefinition(storageTypeInformation);
+
+    var result = serializedObjectIDDataParameterDefinition.CreateDataParameter(commandMock.Object, testValue);
+
+    Assert.That(result, Is.SameAs(dataParameterMock.Object));
+    Assert.That(result.DbType, Is.EqualTo(DbType.AnsiString));
+    Assert.That(result.Value, Is.EqualTo(testValue));
+    Assert.That(result.Size, Is.EqualTo(255));
+  }
+
+  [Test]
+  public void CreateDataParameter_WithNullValue ()
+  {
+    var commandMock = new Mock<IDbCommand>(MockBehavior.Strict);
+    var dataParameterMock = new Mock<IDbDataParameter>(MockBehavior.Strict);
+
+    commandMock
+        .Setup(mock => mock.CreateParameter())
+        .Returns(dataParameterMock.Object);
```

Comment:
If we a) drop the `Verifiable` on the previous test case and b) thus keep the statement single-line, we can to that here, too. simple setup and simple return -> no fuss. 

Noticed / mentioned because the file is new and the style differs be two tests.